src/settings/components/Sets/SetsForm/components/FilteringConditions/FilteringConditions.js

```diff
@@ -89,7 +124,9 @@ const FilteringConditions = ({
               <Field
                 id={`${FILTERING_CONDITIONS_FIELDS.ACTIVE}${index}`}
                 name={`${filteringCondition}.${FILTERING_CONDITIONS_FIELDS.ACTIVE}`}
+                aria-label="filteringConditionActive"
```

Comment:
Could you please clarify the reason you are adding aria-label for fields?

Reply:
For fix axe issue
![image](https://user-images.githubusercontent.com/24813219/93449645-a94bcf00-f8dd-11ea-98c9-ea890e6e396b.png)

Reply:
aria-labels should be readable for the users. The recommended approach, in this case, is to add further translations in the en.json file and then pass the localized result via intl.formatMessage.

Reply:
Fixed